Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+def user_id
+def group_id
+node {
+    user_id = sh(returnStdout: true, script: 'id -u').trim()
+    group_id = sh(returnStdout: true, script: 'id -g').trim()
+}
+
+pipeline {
+    agent {
+        dockerfile {
+            dir '.ops/build-docker'
+            args '-v /var/run/docker.sock:/var/run/docker.sock -v /etc/group:/etc/group'
+            additionalBuildArgs "--build-arg USER_ID=${user_id} --build-arg GROUP_ID=${group_id}"
+        }
+    }
+    parameters {
+        booleanParam(name: 'CREATE_RELEASE_ZIP', defaultValue: false, description: 'Create the release zip from the dist directory')
+        string(name: 'AWS_INSTANCE_TYPE', defaultValue: 'm4.xlarge', description: 'AWS instance type for the databricks, chaos, or release tests (m4.4xlarge or larger is recommended if dataset is >1m)')
+        string(name: 'AWS_INSTANCE_CIDR', defaultValue: '127.0.0.1/32', description: 'The CIDR that will have access to Stardog running on 5820 in the environment for databricks, chaos, or release tests (if applicable)')
```

Comment:
remove these options if we don't need them? fine if you do, just didn't see them used in the PR

Reply:
I have kept a couple of these parameters expecting to use them .